linker/shared/src/main/scala/org/scalajs/linker/analyzer/Infos.scala

```diff
@@ -143,6 +143,21 @@ object Infos {
     final val FlagClassDataAccessed = 1 << 3
     final val FlagStaticallyReferenced = 1 << 4
     final val FlagDynamicallyReferenced = 1 << 5
+
+    /** The class to which we "attach" a given lambda descriptor.
+     *
+     *  It is the class whose `ReachabilityInfoInClass` contains references to
+     *  that descriptor in its `memberInfos`.
+     *
+     *  This is chosen mostly in a way that `jl.Object` does not become a
+     *  central point of contention for all the lambdas in the world.
```

Comment:
Could you expand on this? I do not understand why `NewLambda` needs different handling from `New` in infos (of course in the analysis it does).

Reply:
Ah yes, this is obsolete. Resolving.